src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -175,6 +175,24 @@ public void CDSBeltToggle(boolean reverse) {
     }
   }
 
+  public void CDSBeltToggle(boolean reverse, double beltSpeed) {
```

Comment:
Change the other CDSBeltToggle to just call this one with the constant as the parameter